vms/platformvm/vm_test.go

```diff
@@ -2269,7 +2272,7 @@ func TestTransferSubnetOwnershipTx(t *testing.T) {
 
 func TestBaseTx(t *testing.T) {
 	require := require.New(t)
-	vm, factory, _, _ := defaultVM(t, upgradetest.Latest)
+	vm, factory, _, _ := defaultVM(t, upgradetest.Durango)
```

Comment:
This test makes assertions based on the static fee - which is not compatible with Etna.